app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -130,7 +145,10 @@ def update_classification
 
     classification = classify_single_contention(params)
     Rails.logger.info('Classified 526Submission', id:, saved_claim_id:, classification:, claim_type:)
-    update_form_with_classification_code(classification['classification_code']) if classification.present?
+    return if classification.blank?
+
+    update_form_with_classification_code(classification['classification_code'])
+    classification['classification_code'].present?
```

Comment:
I tweaked the behavior of `update_classification` (and added the `!` to clarify) to return a boolean value of whether or not the contention was classified, as that is a primary eligibility condition for EP 400 merge.